src/Aspire.Dashboard/Components/Pages/Resources.razor.css

```
@@ -1,14 +1,19 @@
 ::deep .resource-state-container {
     display: flex;
-    justify-content: space-between;
     align-items: center;
 }
 
-::deep .resource-error-badge {
+::deep .resource-state-badge {
     padding: 0 3px;
     cursor: pointer;
 }
 
 ::deep fluent-toolbar > h1 {
     padding-left: calc(var(--design-unit) * 1px);
 }
+
+::deep .unread-logs-errors-link::part(control) {
+    padding-right: 0;
+    padding-left: 3px;
```

Looks better on mouse over:
```suggestion
    padding-left: 0;
    margin-left: 3px;
```